src/saving/SaveHelper.cs

```diff
@@ -38,6 +40,18 @@ public enum SaveOrder
         FileSystem,
     }
 
+    /// <summary>
+    ///   Checks whether the last save is made within a timespan of set duration.
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Used for knowing whether to show confirmation dialog on the pause menu when exiting the game.
+    ///     TODO: Implement a method overriding this flag as old in some way.
```

Comment:
Isn't this TODO not necessary, or am I not understanding it correctly?

Reply:
In a scenario where you 1. save the game, 2. return to menu, and 3. start a new game, it may be possible to return to the main menu without the warning to popup (although I haven't got this to yet but still a possibility). So we may want to override the saved recently flag to manually make the popup to appear. This isn't too significant at the moment so it's why I added the TODO here.

Reply:
Ah okay, could you clarify this here?
And this would be pretty easy to make by adding code in MainMenu._Ready to set the last saved time back to null (through a method to not allow any code to set the property).

Reply:
Hang on, that should actually work, I somehow didn't think of setting lastSave to null into account...